ci-operator/jobs/open-cluster-management/kube-state-metrics/OWNERS

```diff
@@ -0,0 +1,11 @@
+# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+- morvencao
```

Comment:
Same comment as on the other OWNERS file.

Reply:
done.